src/assignment/assignment.helper.tsx

```diff
@@ -311,9 +306,13 @@ function getError<T>(rule: ValidationRule<T>, object: T) {
 
 export const getDuplicateTitleOrDescriptionErrors = async (
 	assignment: Partial<Avo.Assignment.Assignment>
-): Promise<string[]> => {
+): Promise<string[] | undefined> => {
 	const errors = [];
 
+	if (!assignment.title || !assignment.description) {
+		return;
```

Comment:
wouldn't it be better to return an empty array? since there are no errors.
```suggestion
		return [];
```

then you can also keep the type: string[]